src/elm/Session/Shared.elm

```diff
@@ -54,6 +56,12 @@ type alias Shared =
     , selectedCommunity : Maybe Eos.Symbol
     , pinVisibility : Bool
     , hasSeenSponsorModal : Bool
+
+    -- Bip39 is what we use to generate the 12 words to login. We store them in
+    -- Shared so that we don't need more HTTP requests than necessary (or to
+    -- store the words hardcoded into Elm code, increasing the bundle size).
+    -- We use Bip39 in the login page to validate the 12 words
+    , bip39 : Bip39Status
```

Comment:
I was about to publish an Elm package including the wordlists, but I think that would've been a lot to load, and we already have them in JS anyway

Reply:
Yeah, not that the word list is too big anyway